docs/get-started/Gsoc.md

```diff
@@ -0,0 +1,96 @@
+---
+id: GSOC
+title: GSOC
+---
+
+
+
+# 2022 GSoC Towhee Proposal
+
+### About Towhee
+
+![Towhee](https://towhee.io/assets/img/logo-title.png)
+
+
+
+Towhee is a flexible, application-oriented framework for running machine learning (ML) pipelines. It is a Python project that aims to make greatly simple anything2vec, allowing everyone - from beginner developers to large organizations - to deploy complex pipelines with just a few lines of code.
+It has become a common practice among the industry to drive data applications with neural network models. Applications involving embedding can be seen in all walks of life, such as product recommender system, copyright protection, data management, software security, new drug discovery, finance, insurance, etc. Despite the fact that the concept of "embed everything" has prevailed in academia and industry in recent years, how to embed unstructured data reasonably and efficiently based on business needs is still an urgent problem that has not yet been resolved. The idea of the Towhee project is to sweep out those obstacles and make MLops significantly easier.
```

Comment:
Neural network models have become the cornerstone of data applications within the industry, with embeddings being seen in all fields, whether it be in product recommendation, copyright flagging, data management, security, pharma, finance, insurance, etc. Despite the concept of "embed everything" being  abundant in academia and industry, there still hasn't been an easy implementation/solution for everyday users. The idea of the Towhee project is to sweep out those obstacles and make MLops significantly easier for all. 


@fzliu  pls check this new version.